package.json

```diff
@@ -21,9 +21,10 @@
     ]
   },
   "scripts": {
+    "dev": "npx nodemon src/index.js -e js,json,graphql --exec babel-node",
     "docker": "docker exec -it zola-node",
     "knex": "npx babel-node --plugins=add-module-exports ./node_modules/.bin/knex --knexfile ./src/server/database/knexfile.js",
-    "dev": "npx nodemon src/index.js -e js,json,graphql --exec babel-node"
+    "test": "jest --verbose --forceExit"
```

Comment:
I should probably document `npm run test -- --watch`? It only run tests that changed.

Reply:
Force exit is needed because db connections are still open and Jest cannot exit :/